docs/source/developing/writing-docs.adoc

```diff
@@ -50,6 +48,51 @@ The latter will start the https://middlemanapp.com[Middleman] server on
 port 4567. You can access the rendered documentation by browsing to
 http://<IP of Docker daemon>:4567.
 
+[[section-deploying-minishift-docs]]
 == Deploying Minishift Documentation
 
-TBD
+The Minishift documentation is deployed on _docs.openshift.org_ under
+link:https://docs.openshift.org/minishift[https://docs.openshift.org/minishift].
+
+To integrate with the _docs.openshift.org_ we are delivering a tarball containing
+the Minishift asciidoc files as well as some link:http://www.asciibinder.org/[AsciiBinder]
+meta data.
+
+This tarball can be build via:
+
+[source,sh]
+----
+$ make gen_adoc_tar
+----
+
+Once the build completes there will be a _minishift-adoc.tar_ under _docs/build_.
+
+How this tarball is integrated into the docs.openshift.org site build is described in
+<<section-building-openshift-docs>>.
+
+[[section-building-openshift-docs]]
+=== Building OpenShift documentation
+
+To start with you need a checkout of the link:https://github.com/openshift/openshift-docs.git[openshift-docs]
+GitHub repository. You need all the tooling to build openshift-docs. Refer to
```

Comment:
Maybe we could break this out into steps:
1. Checkout the link:https://github.com/openshift/openshift-docs.git[openshift-docs]
 +GitHub repository.
2. Install all the tooling required to build the OpenShift-docs. For further information refer to +link:https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/tools_and_setup.adoc[Install and set up the tools and software] section of the openshift-docs repository.
3. You can integrate the Minishift documentation once you are able to successfully run the `rake build` in your openshift-docs checkout.